store/storetest/team_store.go

```diff
@@ -896,18 +896,20 @@ func testTeamMembersWithPagination(t *testing.T, ss store.Store) {
 	store.Must(ss.Team().SaveMember(m4, -1))
 	store.Must(ss.Team().SaveMember(m5, -1))
 
-	r1 = <-ss.Team().GetTeamsForUserWithPagination(uid, 0, 1)
-	require.Nil(t, r1.Err)
-	ms = r1.Data.([]*model.TeamMember)
-	require.Len(t, ms, 1)
+	if result, err := ss.Team().GetTeamsForUserWithPagination(uid, 0, 1); err != nil {
```

Comment:
Nit 1/5 - I think it's cleaner to reduce nesting in tests and write it the same way it was before:
```
result, err := ss.Team().GetTeamsForUserWithPagination(uid, 0, 1);
require.Nil(t, err)
require.Len(t, result, 1)
```